### DISCUSSION TITLE
Ability to apply physics (hinge effect) to the glb/gltf model and move the model to 2D coordinates with the phaser

### DISCUSSION THREAD

**Discussion by sherlockat** [enable3d/enable3d#177] [Help / Questions]
Hello guys,

I'm trying to load a gltf/glb file, apply a hinge effect and then move an object in 2D coordinates.
For example, it is a function that shows the sunglasses glb on the recognized face coordinates.

And the Hinge effect was applied using "enable3d.framework.0.22.0.min.js" library and "PhysicsLoader(ammo/kripken)".

For now I want to move the loaded glb model to 2D coordinates.

I found the following example and tried it.
https://enable3d.io/examples/sync-2d-to-3d.html

However, when I load the "enable3d.phaserExtension.0.22.0.min.js" library, two problems occur.

1. I cannot add a loaded glb object to a scene object.
2. The third.physics.hinge function cannot be executed.

How do I get the Phaser and Physics (for the hinge effect) to work perfectly?
Or is there a better way or example?

**Comment by yandeu** [comment]
To achieve this effect, there is no need for physics nor a hinge constraint. 

If you look at the source code of the example mentioned above, you will see that it is very easy:

```js
// this.rect is from phaser.js
// this.box is from three.js

const x = (this.rect.x / this.cameras.main.width) * 2 - 1
const y = -(this.rect.y / this.cameras.main.height) * 2 + 1

const position = this.third.transform.from2dto3d(x, y, 10)
if (position) {
  const { x, y, z } = position
  this.box.position.set(x, y, z)
}
```
